OrleansDashboard.Hosting/BasicAuthMiddleware.cs

```diff
@@ -1,21 +1,21 @@
-﻿using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Features;
-using System;
+﻿using System;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
 using Microsoft.Extensions.Options;
 
-namespace OrleansDashboard
+namespace OrleansDashboard.Hosting
 {
     internal class BasicAuthMiddleware
     {
-        private readonly RequestDelegate next;
-        private readonly DashboardOptions options;
+        private readonly RequestDelegate _next;
```

Comment:
It seems that you use other code standards. Can you follow the standards from the Dashboard. The PR contains a lot of changes that only contains formattings stuff, such as the whole BasicAuthMiddleware.cs file, Index.html DashboardMiddleware.cs and perhaps even more.

It would be also great to have renames in the commits.

Reply:
Uh right, sorry. I got Resharper setted in automode to correct those stuff, I will revert those renaming asap